formal/codegen-ptx/STATUS.md

```
@@ -0,0 +1,40 @@
+# Status — codegen-ptx formal project
+# Status — codegen-ptx formal project
```

markdownlint-cli2 (0.22.1)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)